src/Dependencies/CodeAnalysis.Debugging/Microsoft.CodeAnalysis.Debugging.nuspec

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata minClientVersion="3.3">
+    <id>$PackageId$</id>
+    <description>$PackageDescription$</description>
+    <version>$Version$</version>
+    <authors>$Authors$</authors>
+    <requireLicenseAcceptance>$RequireLicenseAcceptance$</requireLicenseAcceptance>
+    <licenseUrl>$PackageLicenseUrl$</licenseUrl>
+    <projectUrl>$PackageProjectUrl$</projectUrl>
+    <copyright>$Copyright$</copyright>
+    <developmentDependency>$DevelopmentDependency$</developmentDependency>
+    <tags>$PackageTags$</tags>
+    <serviceable>$Serviceable$</serviceable>
+    <repository type="$RepositoryType$" url="$RepositoryUrl$" commit="$RepositoryCommit$" />
+    <dependencies>
+      <dependency id="Microsoft.CodeAnalysis.PooledObjects" version="$Version$" />
+    </dependencies>
+    <contentFiles>
+      <files include="**/*.cs" buildAction="Compile" />
+    </contentFiles>
+  </metadata>
+  <files>
+    <file src="SourcePackages\Microsoft.CodeAnalysis.Debugging.Package\Microsoft.CodeAnalysis.Debugging.props" target="build" />
+    <file src="$ProjectDirectory$\**\*.cs" target="contentFiles/cs/netstandard1.3" />
+    <file src="$ProjectDirectory$\**\*.cs" target="contentFiles/cs/net45" />
```

Comment:
Shouldn't this be net46?

Reply:
It should not. We need to support `net45` target, so that the sources can be imported to Mono.Cecil.

Reply:
❔ Do we validate the net45 API compatibility during our PR build?

Reply:
Yes, the project is targeting `<TargetFrameworks>netstandard1.3;net45</TargetFrameworks>`